css/style.css

```diff
@@ -1,32 +1,41 @@
-/**
- * Beware that Qt Assistant (QTextBrowser) supports only a limited set of 
- * HTML/CSS: http://qt-project.org/doc/qt-5/richtext-html-subset.html
- */
```

Comment:
As I understood this repo this would not be pushed to the offline manual, if this is not the case this CSS might break it in the Qt browser.